### PR TITLE
Issue/3

### PR DESCRIPTION
**Pull request by MaheshPulle7**
# Pull Request Review

## Checklist

- [x] Issue: #3 
- [x] List of related PRs. : None
- [x] Project builds without any errors.
- [x] Unit Tests updated (or created where needed) and all pass.
- [x] You have ran manual functional "smoke" test (does product as a whole still work?).
- [x] User Story meets the acceptance criteria - and passes. acceptance criteria should be understood at outset.
- [x] There is a PR for update of internal documentation for the affected Project(s) - README etc.
- [x]  **Description of the changes**:  #3

### CODE REVIEW

File: src/main/java/com/ibm/sdwan/viptela/service/impl/KafkaExternalMessagingServiceImpl.java [hunk 1]
```diff
@@ -39,18 +39,14 @@ public KafkaExternalMessagingServiceImpl(SDWDriverProperties properties, KafkaTe
     @Override public void sendExecutionAsyncResponse(ExecutionAsyncResponse request,String tenantId) {
         try {
             final String message = objectMapper.writeValueAsString(request);
-            // ListenableFuture<SendResult<String, String>> future = kafkaTemplate.send(properties.getTopics().getLifecycleResponsesTopic(), message);
```
Comment (lokanalla) [inline src/main/java/com/ibm/sdwan/viptela/service/impl/KafkaExternalMessagingServiceImpl.java:42]: Please remove the commented code

File: src/main/java/com/ibm/sdwan/viptela/utils/ViptelaUtils.java [hunk 2]
```diff
@@ -19,5 +27,52 @@ public static long getDelay(SDWDriverProperties sdwDriverProperties, ExecutionRe
         }
         return vManageApiRetryDelay;
     }
+
+    public static void validateDeploymentProperties(Map<String, Object> deploymentLocationProperties) {
+        if(deploymentLocationProperties==null || deploymentLocationProperties.isEmpty())
```
Comment (lokanalla) [inline src/main/java/com/ibm/sdwan/viptela/utils/ViptelaUtils.java]: format the code here

File: src/main/java/com/ibm/sdwan/viptela/service/impl/KafkaExternalMessagingServiceImpl.java (hunk 1 above)
Comment (MaheshPulle7) [reply src/main/java/com/ibm/sdwan/viptela/service/impl/KafkaExternalMessagingServiceImpl.java:42]: Commented code was removed

File: src/main/java/com/ibm/sdwan/viptela/utils/ViptelaUtils.java (hunk 2 above)
Comment (MaheshPulle7) [reply src/main/java/com/ibm/sdwan/viptela/utils/ViptelaUtils.java]: Code was formatted.

**Review by lokanalla — APPROVED** [review]

**Pull request merged**